.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Scope
+
+
+## To Reproduce
+To reporoduce this issue : 
```

Comment:
reporoduce -> reproduce

As I state in the pull request template, do we need these descriptions?